### ISSUE TITLE
How to find last tensor(operation/name) for inference stage?

### ISSUE THREAD

**Issue by dhlee-jubilee** [melodyguan/enas#86]
Finally, I trained own datasets and save .meta files. 
Now I loaded the trained files and trying to inference on test datasets, but cannot find and load a last logits tensor.

The expected a shape of last tensor should be (1,), due to binary classification, so I used "sess.graph.get_operations()" to find the last tensor.

However, the final model is very complex and an end of tensors' shapes are (50,), (5, 5, 50, 1), (1, 1, 50, 50), (75,), ()  and so on.

I want to find "logits" tensor, and cannot find the last tensor_name. Help me!

(e.g.)
logits = tf.get_default_graph().get_tensor_by_name('save_1/Assign_4430:0')
predictions = sess.run(logits, feed_dict={X: images, y : labels})

**Comment by Nadav1** [comment]
Hi @dhlee-jubilee ,
I suggest you add your own name to the output tensor, just find it in the code and add name='model_output' for example. (Probably the test_preds in the micro child module). You can look for the full name afterwards by printing all tensors in the graph, it will probably be something like:
prefix/Test/model_output:0
